src/status_im/common/scan_qr_code/view.cljs

```diff
@@ -174,10 +174,10 @@
                                                     :on-success-scan set-qr-code-succeeded
                                                     :on-failed-scan  set-rescan-timeout}))}]]
    [hole-view/hole-view
-    {:style style/hole
-     :holes [(assoc qr-view-finder :borderRadius 16)]}
+    {:style [rn/stylesheet-absolute-fill style/hole]
+     :holes [(assoc qr-view-finder :border-radius 16)]}
```

Comment:
@flexsurfer This is an example where we now use kebab-case keys instead of camelCase, if we didn't use camelCase the hole-view just didn't work